deploy/examples/operator.yaml

```diff
@@ -620,7 +620,7 @@ spec:
               value: "5"
 
             - name: ROOK_DISABLE_ADMISSION_CONTROLLER
-              value: "false"
+              value: "true"
```

Comment:
Is this default also in operator-openshift.yaml?

Reply:
@travisn How will this be handled for upgraded clusters? users will just replace the Rook image or they will do kubectl apply -f operator.yaml . should we document this?

Reply:
> Is this default also in operator-openshift.yaml?

there is no env in openshift yaml, which mean `true` but for downstream there is already changes to have it `false`

Reply:
> @travisn How will this be handled for upgraded clusters? users will just replace the Rook image or they will do kubectl apply -f operator.yaml . should we document this?

@Madhu-1 The upgrade guide does just recommend applying a new image version without updating the yaml. I think it will be sufficient if we just put it in the release notes that we recommend disabling this setting. If some users don't change it, it's not critical and they can do it later if the admission controller causes them problems.

Reply:
> > Is this default also in operator-openshift.yaml?
> 
> there is no env in openshift yaml, which mean `true` but for downstream there is already changes to have it `false`

Upstream stream users may be using openshift, so we need to change [this one](https://github.com/rook/rook/blob/master/deploy/examples/operator-openshift.yaml#L619) as well.

Reply:
done